fe/fe-core/src/main/java/org/apache/doris/persist/EditLog.java

```diff
@@ -1576,7 +1576,9 @@ public void logTableRename(TableInfo tableInfo) {
     }
 
     public void logModifyViewDef(AlterViewInfo alterViewInfo) {
-        logEdit(OperationType.OP_MODIFY_VIEW_DEF, alterViewInfo);
+        long logId = logEdit(OperationType.OP_MODIFY_VIEW_DEF, alterViewInfo);
+        LOG.info("log modify view, logId : {}, infos: {}", logId, alterViewInfo);
+        Env.getCurrentEnv().getBinlogManager().addModifyViewDef(alterViewInfo, logId);
```

Comment:
In the replaying stage, the binlog must also be added to the binlog manager. The relevant code is in `loadJournal()`.